client/layout/guided-tours/site-title-tour.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	Next,
+	Quit,
+	Continue,
+	Link,
+	ButtonRow,
+} from 'layout/guided-tours/config-elements';
+import {
+	selectedSiteHasDefaultSiteTitle,
+	userIsOlderThan,
+	isEnabled,
+	userCanEditSettingsOfSelectedSite,
+	isAbTestInVariant,
+} from 'state/ui/guided-tours/contexts';
+import Gridicon from 'components/gridicon';
+
+const TWO_DAYS_IN_MILLISECONDS = 2 * 1000 * 3600 * 24;
+
+export const SiteTitleTour = makeTour(
+	<Tour
+		name="siteTitle"
+		version="20161010"
+		path=""
+		when={ and(
+						isEnabled( 'guided-tours/site-title' ),
+						selectedSiteHasDefaultSiteTitle,
```

Comment:
I think this tour might not be ideal for mobile. We would have trouble positioning "click-settings" step as that is at the bottom of the sidebar and there is no place on sides. It would need to overlap the sidebar and I can see that might be a problem small viewport and cause troubles with scrolling. 

I'd like `isDesktop` approach more. What do you think?